chatbot.py

```diff
@@ -0,0 +1,62 @@
+from chatterbot import ChatBot
+from chatterbot.trainers import ChatterBotCorpusTrainer
+import speech_recognition as sr
+from gtts import gTTS
+import os
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(logging.CRITICAL)
+
+class MyChatBot:
+    
+    def __init__(self, **kwargs):
+        self.chatbot = ChatBot('Robot')
+        #chatbot için yeni bir eğitim oluşturun
+        self.trainer = ChatterBotCorpusTrainer(self.chatbot)
+
+        self.trainer.train("chatterbot.corpus.turkish")
+        # Chatbot'u Türkçe bütüncesine"(corpusuna)"göre eğitin
+        self.trainer.train("chatterbot.corpus.turkish.greetings")
+        # Chatbot'u Türkçe Selamlama bütüncesine"(corpusuna)"göre eğitin
+        self.trainer.train("chatterbot.corpus.turkish.conversations")
+        # Chatbot'u Türkçe Konuşma bütüncesine"(corpusuna)"göre eğitin
+
+        self.recognizer = sr.Recognizer()
+
+    def get_audio_input(self):
+        with sr.Microphone() as source:
+            print("Dinliyorum...")
+            audio = self.recognizer.listen(source)
+
+        try:
+            text = self.recognizer.recognize_google(audio, language='tr-TR')
+            print("Ses Algılandı:", text)
+            return text
+        except sr.UnknownValueError:
+            print("Ses anlaşılamadı")
+            return ""
+        except sr.RequestError:
+            print("Google Speech Recognition servisine şu anda ulaşılamıyor lütfen daha sonra tekrar deneyiniz")
+            return ""
+
+    def speak(self, text):
```

Comment:
The module https://github.com/dmt-labs/modular-biped/blob/develop/modules/tts.py already handles text-to-speech, so you should just be able to publish to the `tts` event and pass a `msg` argument to output.

If this implementation works better than the other, I don't mind you improving that module, but the duplication of functionality can be avoided.